### ISSUE TITLE
Data source auto-discovery and other "import" flow enhancements

### ISSUE THREAD

**Issue by mholt** [timelinize/timelinize#54]
Elaborating and expanding upon thoughts in https://github.com/timelinize/timelinize/issues/53#issuecomment-2338471429... And connected to #3...

As I revise the import flow, it could be useful if data sources could be discovered automatically if they're already on the local system. For example, iPhone backups, iMessage chats, and browser histories (?) are usually in predictable or predefined locations.

We could extend this idea and potentially scan the file system in the background, especially at first run, for existing data. I don't suppose we'd want to match _every_ data source this way, for instance we likely wouldn't want to suggest they import every single image or video file we find. :slightly_smiling_face: But some data sources are pretty obviously data sets they may want to import.

Most people probably won't already have their off-device data consolidated onto their device, however. So we'll probably need an in-app way of showing users how to get their data downloaded in the first place.

Putting all this together, I can foresee some enhancements:

- Inform the user if we've found data sources we recognize (either in predefined locations or by scanning their computer)
- Suggest to the user how they can get more of their data onto their computer. Right now it's not self-evident what kinds of data the app even supports! (Maybe link to tutorials in our docs showing how to download each data source.)

**Comment by rubiojr** [comment]
That'll be quite handy and save users a bunch of work IMO. Reminds me to the Home Assistant devices auto discovery feature.

**Comment by mholt** [comment]
Yeah. The users will still have to do some work when it comes to getting their data downloaded (see #11 -- it's hard to automate), but we can definitely do more once it's on their computer. They might not even know it!

**Comment by mholt** [comment]
The new import flow has been implemented (#3) -- it can now scan a folder tree for data it recognizes, so the rest of this issue, I believe, is strictly UX-related.
